src/shared/api/handle-param-scope.ts

```diff
@@ -0,0 +1,34 @@
+import type { Reactive } from 'vue';
+import { computed, reactive } from 'vue';
+import { useErrorHandlers } from '@/shared/ui/composables/useErrorHandlers';
+import { useErrorRetry, useTask, wheneverRejected } from '@vue-kakuyaku/core';
+
+export function handleParamScope<T, K>(
+  payload: Reactive<T>,
+  fn: (query: Reactive<T>) => Promise<K>,
```

Comment:
```suggestion
  fn: () => Promise<K>,
```

You can include the payload in the closure naturally, without needing to introduce any arguments here:

```ts
handleParamScope(payload, http.fetchTransactions)
// vs
handleParamScope(() => http.fetchTransactions(payload))
```